### PR TITLE
Adding decarad service and implementing stepper frequency tuning

### PR DESCRIPTION
**Pull request by lisazacarias**

### CODE REVIEW

File: decarad_service/decarad_service.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        self.add_pvs(decaradPVs)
+
+
+def main():
+    service = DecaradService()
+    asyncio.get_event_loop()
+    _, run_options = ioc_arg_parser(
+            default_prefix='',
+            desc="Simulated Decarad")
+    run(service, **run_options)
```
Comment (Derikka) [inline decarad_service/decarad_service.py:47]: General question: What is this decarad_service.py being used for? When someone says "decarad," I imagine the things you guys have to move down in the linac west tunnel. I thought that was a physical object having something to do with radiation (?). Do readings from this decarad actually get passed to EPICS (and therefore requiring us to make a simulacrum equivalent file?)

File: sc_rf_service/sc_rf_service.py [hunk 2]
```diff
@@ -204,12 +272,33 @@ class CavityPVGroup(PVGroup):
                                                        "PARKED"))
     cudDesc: PvpropertyChar = pvproperty(value="Name", name="CUDDESC",
                                          dtype=ChannelType.CHAR)
-
-    def __init__(self, prefix, length):
-
+    ssa_latch: PvpropertyEnum = pvproperty(value=1, name="SSA_LTCH",
+                                           dtype=ChannelType.ENUM,
+                                           enum_strings=("OK", "Fault"))
```
Comment (Derikka) [inline sc_rf_service/sc_rf_service.py:277]: What's the difference between this ssa_latch fault and the one at the beginning of `class CavFaultPVGroup(PVGroup):`? I'm wondering if this is a duplicate of line 156:

latch: PvpropertyEnum = pvproperty(value=1, name="SSA_LTCH",
                                       dtype=ChannelType.ENUM,
                                       enum_strings=("OK", "Fault"))

File: sc_rf_service/sc_rf_service.py [hunk 3]
```diff
@@ -204,12 +272,33 @@ class CavityPVGroup(PVGroup):
...
+    detune_rfs: PvpropertyInteger = pvproperty(value=0, name="DF",
+                                               dtype=ChannelType.INT)
+    step_temp: PvpropertyFloat = pvproperty(value=35.0, name="STEPTEMP",
+                                            dtype=ChannelType.FLOAT)
+    # TODO probe the PV for actual enum strings
+    fscan_stat: PvpropertyEnum = pvproperty(name="FSCAN:SEARCHSTAT",
+                                            value=0, dtype=ChannelType.ENUM,
+                                            enum_strings=("No faults",
+                                                          "faulted"))
+
```
Comment (Derikka) [inline sc_rf_service/sc_rf_service.py:291]: I probed "ACCL:L2B:1020:FSCAN:SEARCHSTAT" as an example and I believe these are the enum strings you want - 

`enum_strings=("No errors", "None found", "Unknown mode", "Wrong freq", "Data nonsync")`

File: decarad_service/decarad_service.py (hunk 1 above)
Comment (lisazacarias) [reply decarad_service/decarad_service.py:47]: They are physical objects that read out into EPICS!

File: sc_rf_service/sc_rf_service.py (hunk 2 above)
Comment (lisazacarias) [reply sc_rf_service/sc_rf_service.py:277]: Ah good catch! I think that happened when I tried to resolve some merge conflicts...

File: sc_rf_service/sc_rf_service.py (hunk 3 above)
Comment (lisazacarias) [reply sc_rf_service/sc_rf_service.py:291]: You are the best, hands down

File: decarad_service/decarad_service.py (hunk 1 above)
Comment (Derikka) [reply decarad_service/decarad_service.py:47]: Ah, got it. Thank you:D

**Review by Derikka — APPROVED** [review]

**Pull request merged**